src/components/Book.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/jsx-one-expression-per-line */
```

Comment:
- [ ] Let's not disable linter rules, this should be the last resource as it defeats the purpose of linter rules themselves, which is to keep the code clean, readable, and standardized across teammates, coworkers, etc.

Instead, let's find a solution to this error on `line 19`. Tip: The rule indicates that the line in question should be split into multiple ones.

Reply:
fixed